src/common/components/table/table-examples/TxTableCellRenderers.tsx

```diff
@@ -236,58 +292,42 @@ const StatusTag = ({ status }: { status: TransactionStatus | MempoolTransactionS
 };
 
 export const TransactionTitleCellRenderer = (value: TxTableTransactionColumnData) => {
-  const { functionName, contractName, txType, status, amount, tenureChangePayload, smartContract } =
-    value;
+  const {
+    functionName,
+    txType,
+    status,
+    amount,
+    tenureChangePayload,
+    smartContract,
+    txId,
+    blockHeight,
+  } = value;
 
-  let content: React.ReactNode = null;
-  if (txType === 'contract_call') {
-    content = (
-      <Flex gap={1}>
-        <Text fontSize="sm" fontWeight="medium" color="textPrimary" whiteSpace="nowrap">
-          {functionName}
-        </Text>
-        <Flex gap={1} alignItems="center" bg="surfacePrimary" borderRadius="md" px={1.5} py={0.5}>
-          <Icon h={3} w={3} color="iconSecondary">
-            <ClarityIcon />
-          </Icon>
-          <Text fontSize="xs" fontWeight="medium" color="textSecondary" whiteSpace="nowrap">
-            {getContractName(contractName ?? '')}
-          </Text>
-        </Flex>
-      </Flex>
-    );
-  }
-  if (txType === 'token_transfer') {
-    content = (
-      <Text fontSize="sm" fontWeight="medium" color="textPrimary">
-        {amount} STX
-      </Text>
-    );
-  }
+  const text =
+    txType === 'contract_call'
+      ? functionName
+      : txType === 'token_transfer'
+        ? `Send ${amount} STX`
+        : txType === 'tenure_change'
+          ? `Tenure ${tenureChangePayload?.cause} (#${blockHeight})`
+          : txType === 'coinbase'
+            ? `Block #${blockHeight}`
+            : txType === 'smart_contract'
+              ? getContractName(smartContract?.contractId ?? '')
+              : '';
```

Comment:
instead of deep nesting I suggest creating a dictionary to map tx type to text, e.g.
```
const txTextMap = {
  contract_call: (data: TxTableTransactionColumnData) => data.functionName,
  token_transfer: (data: TxTableTransactionColumnData) => `Send ${data.amount} STX`,
  tenure_change: (data: TxTableTransactionColumnData) => `Tenure ${data.tenureChangePayload?.cause} (#${data.blockHeight})`,
  coinbase: (data: TxTableTransactionColumnData) => `Block #${data.blockHeight}`,
  smart_contract: (data: TxTableTransactionColumnData) => getContractName(data.smartContract?.contractId ?? ''),
};
```